lib/phonemiser.py

```diff
@@ -153,44 +155,45 @@ def phonemise(token, lang):
         lang = iso_3to2[lang]
         return lookup_tables[lang].transliterate(token)
     if lang in ["ab", "abk"]:
-        return maxphon(lookup_tables["ab"], token)
+        return _maxphon(lookup_tables["ab"], token)
     if lang in ["as", "asm"]:
-        return maxphon(lookup_tables["as"], token)
+        return _maxphon(lookup_tables["as"], token)
     if lang in ["br", "bre"]:
-        return maxphon(lookup_tables["br"], token)
+        return _maxphon(lookup_tables["br"], token)
     if lang in ["cv", "chv"]:
-        return maxphon(lookup_tables["cv"], token)
+        return _maxphon(lookup_tables["cv"], token)
     if lang in ["cy", "cym"]:
-        return maxphon(lookup_tables["cy"], token)
+        return _maxphon(lookup_tables["cy"], token)
     if lang in ["dv"]:
-        return maxphon(lookup_tables["dv"], token)
+        return _maxphon(lookup_tables["dv"], token)
     if lang in ["el", "ell"]:
-        return maxphon(lookup_tables["el"], token)
+        return _maxphon(lookup_tables["el"], token)
     if lang in ["fi", "fin"]:
-        return maxphon(lookup_tables["fi"], token)
+        return _maxphon(lookup_tables["fi"], token)
     if lang in ["ja", "jpn"]:
-        return jpn(token)
+        return _jpn(token)
     if lang in ["mn", "mon"]:
-        return maxphon(lookup_tables["mn"], token)
+        return _maxphon(lookup_tables["mn"], token)
     if lang in ["or", "ori"]:
-        return maxphon(lookup_tables["or"], token)
+        return _maxphon(lookup_tables["or"], token)
     if lang in ["quc"]:
-        return maxphon(lookup_tables["quc"], token)
+        return _maxphon(lookup_tables["quc"], token)
     if lang in ["tr", "tur"]:
-        return maxphon(lookup_tables["tr"], token)
+        return _maxphon(lookup_tables["tr"], token)
     if lang in ["tt", "tat"]:
-        return maxphon(lookup_tables["tt"], token)
+        return _maxphon(lookup_tables["tt"], token)
     if lang in ["sah"]:
-        return maxphon(lookup_tables["sah"], token)
+        return _maxphon(lookup_tables["sah"], token)
     if lang in ["ur", "urd"]:
-        return maxphon(lookup_tables["ur"], token)
+        return _maxphon(lookup_tables["ur"], token)
     if lang.startswith("zh-"):
         return lookup_tables[lang].transliterate(token)
 
     return token
 
 
 def init():
+    """Init all the phonemisers."""
```

Comment:
This is a weird convention and not very pythonic.  We'll clean this up later though so leave it for now.

Reply:
Not sure what to do there. Can I resolve this?

Reply:
You can but it's just a matter of dropping this initialization or encoding it into a constructor this type.